content/2021-08-26-drg-trust/index.md

```diff
@@ -0,0 +1,165 @@
++++
+title = "Using X509 Client certificates for devices authentication with drogue-cloud"
+extra.author = "jb_and_vedang"
++++
+
+This article shows how you can setup X.509 certificates to authenticate devices connecting to Drogue Cloud and an example of how [`drg`](https://github.com/drogue-iot/drg) makes it easy.
+
+<!-- more -->
+
+# X509 Authentication?
+
+If you are already master how chains of trust works with X509 certs feel free to skip to the next part. Here is a quick explanation otherwise :
```

Comment:
```suggestion
If you are already master how chains of trust works with X509 certs feel free to skip to the next part. Here is a quick explanation otherwise:
```